argilla/CHANGELOG.md

```diff
@@ -14,7 +14,19 @@ These are the section headers that we use:
 * "Security" in case of vulnerabilities.
 -->
 
-## [2.0.0rc1](https://github.com/argilla-io/argilla/compare/v1.29.0...v2.0.0rc)
+## [Unreleased]()
+
+## [2.0.0](https://github.com/argilla-io/argilla/compare/v2.0.0rc1...v2.0.0)
+
+- Added core class refactors. For an overview, see [this blog post](https://argilla.io/blog/introducing-argilla-new-sdk/)
```

Comment:
@burtenshaw please can you organize the changes into different sections as we usually do? 

We use the sections defined at https://keepachangelog.com/en/1.1.0